Source/WebCore/css/parser/CSSTokenizer.h

```diff
@@ -49,12 +49,15 @@ class CSSTokenizer {
     static std::unique_ptr<CSSTokenizer> tryCreate(const String&);
     static std::unique_ptr<CSSTokenizer> tryCreate(const String&, CSSParserObserverWrapper&); // For the inspector
 
-    explicit CSSTokenizer(const String&);
+    WEBCORE_EXPORT explicit CSSTokenizer(const String&);
     CSSTokenizer(const String&, CSSParserObserverWrapper&); // For the inspector
 
-    CSSParserTokenRange tokenRange() const;
+    WEBCORE_EXPORT CSSParserTokenRange tokenRange() const;
     unsigned tokenCount();
 
+    static bool isNewline(UChar);
+    static bool isWhitespace(CSSParserTokenType);
```

Comment:
This function would be easier to use as a non-member function and since it takes an argument of a specific type it would be unambiguous and would overload with other similarly named functions. So I would be tempted to make it a non-member. But it’s OK either way.

Reply:
made both of them non member functions.